src/main/resources/assets/computercraft/lua/rom/programs/echo.lua

```diff
@@ -0,0 +1 @@
+print( ... )
```

Comment:
If variables exist, the ability to echo them provides the ability to see their values, which is valuable.
However, that points out a big missing piece in this PR: Currently there's no way to set variables from the shell!